pkg/storkctl/schedulepolicy.go

```diff
@@ -161,18 +160,18 @@ func newCreateSchedulePolicyCommand(cmdFactory Factory, ioStreams genericcliopti
 				util.CheckErr(err)
 				return
 			}
-			msg := fmt.Sprintf("SchedulePolicy %v created successfully", schedulePolicy.Name)
+			msg := fmt.Sprintf("schedule policy %v created successfully", schedulePolicy.Name)
 			printMsg(msg, ioStreams.Out)
 		},
 	}
-	//Picking up user inputs and setting the defaults for flags
-	createSchedulePolicyCommand.Flags().StringVarP(&schedulePolicyType, "policy-type", "t", "Interval", "Select Type of Schedule Policy to apply. Interval / Daily / Weekly / Monthly. ")
-	createSchedulePolicyCommand.Flags().IntVarP(&intervalMinutes, "interval-minutes", "", 30, "Specify the interval, in minutes, after which Portworx should trigger the operation. ")
-	createSchedulePolicyCommand.Flags().IntVarP(&retain, "retain", "", 0, "For backup operations, specify how many backups Portworx should retain. ")
-	createSchedulePolicyCommand.Flags().StringVarP(&time, "time", "", "12:00AM", "Specify the time of the day in the 12 hour AM/PM format, when Portworx should trigger the operation. ")
-	createSchedulePolicyCommand.Flags().StringVarP(&dailyForceFullSnapshotDay, "force-full-snapshot-day", "", "MONDAY", "For daily scheduled backup operations, specify on which day to trigger a full backup. ")
-	createSchedulePolicyCommand.Flags().StringVarP(&weeklyDay, "weekly-day", "", "Sunday", "Specify the day of the week when Portworx should trigger the operation. You can use both the abbreviated or the full name of the day of the week")
-	createSchedulePolicyCommand.Flags().IntVarP(&monthlyDate, "monthly-date", "", 1, "Specify the day of the month when Portworx should trigger the operation")
+	// Picking up user inputs and setting the defaults for flags
+	createSchedulePolicyCommand.Flags().StringVarP(&schedulePolicyType, "policy-type", "t", "Interval", "Select Type of schedule policy to apply. Interval / Daily / Weekly / Monthly.")
+	createSchedulePolicyCommand.Flags().IntVarP(&intervalMinutes, "interval-minutes", "i", 30, "Specify the interval, in minutes, after which Portworx should trigger the operation.")
```

Comment:
Have added a shorthand for interval-minutes (-i) since Interval Schedule Policy seems to be most commonly used by actual customers